tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -231,23 +225,38 @@ def test(
             run_connector_test_pipeline,
             "Test Pipeline",
             ctx.obj["concurrency"],
+            ctx.obj["dagger_logs_path"],
             ctx.obj["execute_timeout"],
         )
-
     except Exception as e:
-        click.secho(str(e), err=True, fg="red")
+        main_logger.error("An error occurred while running the test pipeline", exc_info=e)
         update_global_commit_status_check_for_tests(ctx.obj, "failure")
         return False
-    global_success = all(connector_context.state is ContextState.SUCCESSFUL for connector_context in connectors_tests_contexts)
-    update_global_commit_status_check_for_tests(ctx.obj, "success" if global_success else "failure")
+    finally:
+        # We always want to save the dagger logs, even if the pipeline failed.
```

Comment:
👍

Reply:
Idea: Given this should we hoist this logic up to the connector group so publish and build can benefit?

Reply:
Nice suggestion, I moved this logic to https://github.com/airbytehq/airbyte/blob/e6a7b7d6c04e22c6c565a15f0ce0467d5f6d69bf/tools/ci_connector_ops/ci_connector_ops/pipelines/utils.py#L334